habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -141,16 +144,17 @@ def get_walk_pose(
         distance_to_walk = np.linalg.norm(forward_V)
         did_rotate = False
 
+        # The angle we initially want to go to
+        new_angle = np.arctan2(forward_V[0], forward_V[2]) * 180.0 / np.pi
```

Comment:
Should we create a variable 
```
deg_per_rads = 180.0 / np.pi
```
and reuse it throughout. Would that increase clarity ?

Reply:
Thanks for the suggestion, modified it!